demo/kubernetes/rook-pvcontroller.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: v1
```

Comment:
This file would help us get over the RBD tools dependency. Once we get our external volume plugin, i hope this is not needed.